### ISSUE TITLE
ensemble plot_envelope legend

### ISSUE THREAD

**Issue by CommonClimate** [LinkedEarth/Pyleoclim_util#400]
**Is your feature request related to a problem? Please describe.**
by default the legend has 1 column, which tends to interfere with the plotted envelopes (e.g. for LMR GMST) 
**Describe the solution you'd like**
set `lgd_kwargs={'ncol':3,'loc':'upper center'}` as default, for a better look:
![envelope_legend](https://user-images.githubusercontent.com/13760223/234998555-0f177e98-7a41-4ceb-8705-782fd4e5d02f.png)

**Additional context**
- Probably not needed for MultiplePSD, because most spectra a red, so the 1-column model on the upper right works great there. 
- Make sure to update L0_basic_MSES_manipulation.ipynb once this is done to remove the unnecessary legend customization.
